.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  schedule:
+    - cron: 0 2 * * 1-5
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  tests:
+    name: Tests PHP ${{ matrix.php }} ${{ matrix.dependency }} (Symfony ${{ matrix.symfony }})
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}
```

Comment:
Using docker packages like this seams a bit more complicated. What about just using `shivammathur/setup-php@2.7.0`?